popup.html

```diff
@@ -23,5 +21,6 @@ <h1 style="margin-bottom: 0.2em">No More Domains</h1>
     <div class="power-switch">
       <button onclick="onOff()">Toggle NMD</button>
     </div>
+    <script src="blockpage.js" type="text/javascript"></script>
```

Comment:
While this isn't important in a chrome extension I will merge as it is best practice